coverage/PENDING_REVIEW/intel/0456-COVERAGE-brw-fs-2792/cov-set-output-color-function-call-nested-loop.amber

```diff
@@ -0,0 +1,244 @@
+#!amber
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# A test for a coverage-gap found by the GraphicsFuzz project.
+
+# Short description: A fragment shader that covers specific BRW code paths
+
+# The test passes because the shader always writes red.
+
+# Optimized using spirv-opt with the following arguments:
+# '-O'
+# spirv-opt commit hash: a0370efd589be33d5d9a85cfde2f85841b3755af
+
+
+
+SHADER vertex variant_vertex_shader PASSTHROUGH
+
+# variant_fragment_shader is derived from the following GLSL:
+# #version 320 es
+# #define _int_0 _GLF_uniform_int_values[0]
+# #define _int_1 _GLF_uniform_int_values[1]
+# #define _int_14 _GLF_uniform_int_values[2]
+# #define _int_5 _GLF_uniform_int_values[3]
+# #define _int_10 _GLF_uniform_int_values[4]
+# #define _int_13 _GLF_uniform_int_values[5]
+#
+# precision highp int;
+# precision highp float;
+#
+# // Contents of _GLF_uniform_int_values: [0, 1, 14, 5, 10, 13]
+# layout(set = 0, binding = 0) uniform buf0
+# {
+#     int _GLF_uniform_int_values[6];
+# };
+# layout(location = 0) out vec4 _GLF_color;
+#
+# int func (int x)
+# {
+#     if (x > _int_5)
+#     {
+#         return 2;
+#     }
+#
+#     vec4 v;
+#
+#     for (int i = _int_0; i < _int_10; i++)
+#     {
+#         v = vec4(float(x + i));
+#     }
+#
+#     _GLF_color = v;
+#
+#     return _int_1;
+# }
+#
+# void main()
+# {
+#     int a = _int_0;
+#
+#     for (int i = _int_1; i < _int_10; i++)
+#     {
+#         a += func(i);
```

Comment:
Worth saying that `a` gets the value 13 because `a` returns 1 five times and 2 four times, and `_GLF_color` gets the value 14 due to `func` is called with `i == 5`.